Formula/ioctl.rb

```diff
@@ -0,0 +1,30 @@
+#------------------------------------------------------------------------------
+# ioctl.rb
+#
+# Homebrew formula for ioctl.  Homebrew (http://brew.sh/) is
+# the de-facto standard package manager for OS X, and this Ruby script
+# contains the metadata used to map command-line user settings used
+# with the 'brew' command onto build options.
+#
+# Copyright (c) 2019 IoTeX Foundation
+#------------------------------------------------------------------------------
+class Ioctl < Formula
+  desc "Iotex ctl is a command-line interface for interacting with the IoTeX blockchain"
+  homepage "https://docs.iotex.io/developer/get-started/ioctl-install.html"
+  url "https://github.com/iotexproject/iotex-core/archive/v1.1.1.tar.gz"
+  sha256 "fcff5ce4231253ebbe04e0405a36a7b89fb2d6c1b30ab0b5deac6ac6f84f9f8b"
+  license "Apache-2.0"
+
+  depends_on "go"
```

Comment:
Is `go` needed at runtime?

Reply:
As a compiled language, golang does not need to be dependent at runtime. I may not know the operating mechanism of brew, because I released the source code, so I think it is needed on the client. I don't know what stage the installation belongs to on the client, and I am afraid that brew cleans up the non-build stage installation, because our target customers are dependent on golang, and golang generally does not need to be deleted, for fear that brew will delete it by mistake.

Reply:
brew doesn't delete anything unless you tell it to.

Reply:
I has add " => :build" already